src/js/components/DataFilters/stories/Layer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Box, Data, DataFilter } from 'grommet';
+
+import { DataFilters } from '../DataFilters';
+import { DATA } from '../../DataTable/stories/data';
+
+export const Layer = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box fill align="center" justify="start" pad="large">
+    <Data data={DATA}>
+      <DataFilters layer>
+        <DataFilter property="location" />
```

Comment:
```suggestion
        <DataFilter property="location" contentProps={{ width: 'medium' }} />
```
We've been defaulting to medium width for formfields.

Reply:
Does DataFilter pass FormField props through?

Reply:
DataFilter does pass `contentProps` through.

However, the desired behavior feels like something that should be built in or theme driven so that the caller doesn't need to pass this to every DataFilter. I'm actually wondering if we should build this into the theme at `theme.formField.content.width`. Thoughts?

Reply:
I like that idea, it's a lot of repeated implementation to have to do it each time.